frontend/packages/kubevirt-plugin/src/components/modals/vm-flavor-modal/vm-flavor-modal.tsx

```diff
@@ -43,25 +37,25 @@ import { validateFlavor } from '../../../utils/validations/vm/flavor';
 import { isValidationError } from '../../../utils/validations/common';
 import { useShowErrorToggler } from '../../../hooks/use-show-error-toggler';
 import { getDialogUIError } from '../../../utils/strings';
+import { flavorSort } from '../../../utils/sort';
+import { getTemplateFlavors } from '../../../selectors/vm-template/advanced';
+import { getVMTemplateNamespacedName } from '../../../selectors/vm-template/selectors';
 
 const getId = (field: string) => `vm-flavor-modal-${field}`;
 
+const getFlavors = (template: TemplateKind) => {
```

Comment:
can we change the name so we will not mistake it with the global 'getFlavors' one ?
`getTempalteSortedFlavors` ?

Reply:
we can do it ia a followup ...